NEP-P2P_Protocol_change.mediawiki

```diff
@@ -0,0 +1,129 @@
+<pre>
+  NEP: <to be assigned>
+  Title: P2P Protocol change
+  Author: Shargon and Belane
+  Type: Standard
+  Status: Draft
+  Created: 2018-03-07
+</pre>
+
+==Abstract==
+
+We believe that a P2P protocol optimization is necessary in order to avoid unnecessary data exchange between nodes.
+In this sense, reducing the computational consumption and, consequently, improving and speeding up the communication between nodes.
+
+==Motivation==
+
+Improve the P2P protocol to support higher loads, as well as improving the communication speed and prioritizing consensus messages. 
+In addition, we think that it is necessary to implement a compression mechanism and save all bytes that should be possible.
+
+==Specification==
+
+TODO
+
+==Rationale==
+
+TODO
+
+==Backwards Compatibility==
+
+TODO
+
+==Test Cases==
+
+TODO
+
+==Implementation==
+
+The current header includes a magic number of 4 bytes in all messages. 
+With this current proposal, this header is sent only during the initial negotiation of the communication, avoiding message overload.
```

Comment:
```suggestion
With this proposal, this header is sent only during the initial negotiation of the communication, reducing overhead in all remaining communication messages.
```
"current proposal" suggests _this_ one will change. At best it will be amended or superseded so I think it is better to refer to it as "This proposal".

I had to read "avoiding message **overload**" a couple of times before I understood the intention. I think this suggestion is more self explanatory/clear.